### PR TITLE
Added Turkish names and date format

### PR DESCRIPTION
**Pull request by aykalkan**
Added Turkish hijri month and day names. Changed "fullDate" method to implement Turkish date presentation format.

### CODE REVIEW

File: lib/hijri_calendar.dart [hunk 1]
```diff
@@ -314,7 +320,9 @@ class HijriCalendar {
   List<int?> toList() => [hYear, hMonth, hDay];
 
   String fullDate() {
-    return format(hYear, hMonth, hDay, "DDDD, MMMM dd, yyyy");
+    String dateFormat = "DDDD, MMMM dd, yyyy";
+    if (language == 'tr') dateFormat = "dd MMMM yyyy DDDD";
```
Comment (iballan) [inline lib/hijri_calendar.dart]: Instead of initializing it twice, you could make it like this: 
`String dateFormat = (language == 'tr') ? "dd MMMM yyyy DDDD" : "DDDD, MMMM dd, yyyy";`

File: lib/hijri_calendar.dart [hunk 2]
```diff
@@ -218,49 +224,49 @@ class HijriCalendar {
...
           "DD", "${_local[language]!['short_days']![wkDay ?? weekDay()]}");
     }
 
     //============== Month ========================//
     // 1
-    if (newFormat.contains("mm")) {
+    if (format.contains("mm")) {
       newFormat = newFormat.replaceFirst("mm", monthString);
-    } else {
+    } else if (format.contains("m")){
```
Comment (iballan) [inline lib/hijri_calendar.dart]: It would be great if you add some tests to you PR, so you can avoid any untested edge case, specially after adding "m" check.

**Review by iballan — CHANGES_REQUESTED** [review]

File: lib/hijri_calendar.dart (hunk 1 above)
Comment (aykalkan) [reply lib/hijri_calendar.dart]: Ok. My bad. Will do.

File: lib/hijri_calendar.dart (hunk 2 above)
Comment (aykalkan) [reply lib/hijri_calendar.dart]: So do you suggest writing a final `else` to cover unpredicted use cases? Or do you mean wring a test code that checks possible errors in the package?

Comment (iballan) [reply lib/hijri_calendar.dart]: It would be great if you cover the else preventing any unpredicted case,
Regarding the tests, there are already many tests are written in the package already, you may take it as an example and write similar tests.

**Review by ahmedoid — APPROVED** [review]

File: lib/hijri_calendar.dart (hunk 2 above)
Comment (aykalkan) [reply lib/hijri_calendar.dart]: I didn't apply these changes because they are not relevant with Turkish language support. I'll happily contribute with another PR for these fixes.
